backend/Models/TimeSlot.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CinemaApp.Models
+{
+    public class TimeSlot
+    {
+        public int ID { get; set; }
+        public DateTime Time { get; set; }
+        public CinemaRoom CinemaRoom { get; set; }
```

Comment:
Are CinemaRoom and Movie classes added to the commit?

Reply:
No, they were not my responsibility. But I made them in my local project in order to define these classes. I just did not commit them because I though that it is somebody else's task.

Reply:
Done

Reply:
> 
> 
> No, they were not my responsibility. But I made them in my local project in order to define these classes. I just did not commit them because I though that it is somebody else's task.

If you commit your changes and CinemaRoom is not part of the final project, it's just local on your computer, how do you think the project will build? What if there was an automated build process?

Reply:
> Hello - how is this going ? @hasdarius please fix with the comments from @danbris

Working on it.

Reply:
> > No, they were not my responsibility. But I made them in my local project in order to define these classes. I just did not commit them because I though that it is somebody else's task.
> 
> If you commit your changes and CinemaRoom is not part of the final project, it's just local on your computer, how do you think the project will build? What if there was an automated build process?

I know it wouldn't have built unless the others uploaded their part, but is it right for me to commit their tasks?

Reply:
But is it right for you to break the build, to make the application unusable? In this case you should communicate with the other "responsible" person and see who needs to do what.

Reply:
> Are CinemaRoom and Movie classes added to the commit?

Movie is added